<launch>

<arg name="rgb_image" value="camera/rgb/image_color"/>
<arg name="depth_image" value="camera/depth/image"/>

<!-- Display the current rgb image-->

<node name="input_viewer" pkg="image_view" type="image_view">
    <remap from="image" to="$(arg rgb_image)" />
    <param name="autosize" value="False" />
  </node>

<!-- Display the starting rgb image with projected outline-->

<node name="output_viewer" pkg="image_view" type="image_view">
    <remap from="image" to="rgb_image_out" />
    <param name="autosize" value="False" />
  </node>


<!-- Display the current depth image-->

<node name="depth_input_viewer" pkg="image_view" type="image_view">
    <remap from="image" to="$(arg depth_image)" />
    <param name="autosize" value="False" />
  </node>

<!-- Display the starting depth image with projected outline-->

<node name="depth_output_viewer" pkg="image_view" type="image_view">
    <remap from="image" to="depth_image_out" />
    <param name="autosize" value="False" />
  </node>

<node name="depth_output_viewer2" pkg="image_view" type="image_view">
    <remap from="image" to="depth_image_out2" />
    <param name="autosize" value="False" />
  </node>

</launch>
